cmd/notification.go

```diff
@@ -46,7 +47,13 @@ func (sys *NotificationSys) GetARNList() []string {
 	arns := []string{}
 	region := globalServerConfig.GetRegion()
 	for _, targetID := range sys.targetList.List() {
-		arns = append(arns, targetID.ToARN(region).String())
+		// httpclient target is part of ListenBucketNotification
+		// which doesn't need to be listed as part of the ARN list
+		// This list is only meant for external targets, filter
+		// this out pro-actively.
+		if !strings.HasPrefix(targetID.String(), "httpclient") {
```

Comment:
Use `targetID.ID` and check for `httpclient+` because ID of any target is free formed string.  config may have id `httpclient`

Reply:
Sure let me do that.